excel_import_export/models/ir_report.py

```diff
@@ -41,4 +39,4 @@ def _get_report_from_name(self, report_name):
             ("report_name", "=", report_name),
         ]
         context = self.env["res.users"].context_get()
-        return report_obj.with_context(context).search(conditions, limit=1)
+        return report_obj.with_context(context=context).search(conditions, limit=1)
```

Comment:
```suggestion
        return report_obj.with_context(**context).search(conditions, limit=1)
```
Why do we even need this `context`???
Don't we already have one?
Are we missing something?

Reply:
I think the context may be removed.